lib/presets.js

```diff
@@ -48,6 +54,7 @@ module.exports = {
   babel,
   babelPreserveModules,
   'babel-preserve-modules': babelPreserveModules,
+  'babel-typescript': babelTypescript,
```

Comment:
Good choice to make this additive instead rather than part of the default `babel` :)